frontend/lib/norent/components/logo.tsx

```diff
@@ -24,7 +25,7 @@ export const NorentLogo = (props: {
           ? "logo-dark"
           : "logo"
       )}
-      alt="No Rent"
+      alt={props.altText || "No Rent"}
```

Comment:
This is out of scope since it's not a localized string, but I am super confused about when we're supposed to write "NoRent" vs. "No Rent" vs. "No rent".

Reply:
Hmmm so my impression is we should always refer to the tool/product as "NoRent". The only reason I split it into two words here is so a screenreader would pronounce it properly when reading it out loud.